tool/build.sh

```diff
@@ -2,18 +2,31 @@ ORIG=`pwd`
 DIR="$( cd "$( dirname "${BASH_SOURCE[0]}" )" >/dev/null 2>&1 && pwd )"
 ROOT=$DIR/../
 PACKAGE_VERSION=`git describe --tags`
+REPM_VERSION='3459bfd48b85baf7df7d7332e64ae5c3ac2e3320'
 
 echo "Building Flutter SDK..."
 
 cd $ROOT
 set -x
 
-# Copy the local repo into the output dir
 rm -rf build
 mkdir build
 cd build
+
+# Copy the local repo into the output dir
 mkdir replicache-flutter-sdk
 ls ../ | grep -v build | grep -v sample | grep -v tool | grep -v gh-dl-release | grep -v doc | xargs -I{} cp -R ../{} replicache-flutter-sdk/{}
 
+# Build repm
+git clone https://github.com/rocicorp/replicache-client
+cd replicache-client
```

Comment:
[fyi] You could use `(` so you do not have to undo the `cd`

Reply:
I don't understand -- `(` ?

Reply:
Not a big deal. I was just pointing out that the shell script can be simplified slightly by doing something like:

```bash
(
  cd x
  doThing()
)
```